src/main/java/zmaster587/advancedRocketry/entity/EntityRocket.java

```diff
@@ -1426,14 +1419,13 @@ private void reachSpaceUnmanned()
 				}
 
 				this.setPositionAndUpdate(pos.x, getEntryHeight(destinationDimId), pos.z);
-				return;
 			}
 			else {
 
 				//Make player confirm deorbit if a player is riding the rocket
 				if(hasHumanPassenger()) {
 					setInFlight(false);
-					pos.y = (float) getEntryHeight(destinationDimId);
+					this.setPositionAndUpdate(this.posX, getEntryHeight(destinationDimId), this.posZ);
```

Comment:
IDEA said this originally caused an NPE due to dereferencing `pos`. I hope my replacement doesn't change the functionality from what was intended